e2e/tests/definitions.js

```diff
@@ -1,143 +1,150 @@
 // Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
 // SPDX-License-Identifier: MIT
 import { definitionsMap } from '../maps/definitions'
+import { setDefaultOptions } from 'expect-puppeteer'
+
 const puppeteer = require('puppeteer')
+const defaultTimeout = process.env.JEST_TIMEOUT ? process.env.JEST_TIMEOUT : 30000
```

Comment:
This value needs to be set in these 3 points:
- puppeteer configuration
- jest configuration
- test callback

Otherwise, the entire test will fail.